vm/src/test/scala/pravda/vm/VmSandbox.scala

```diff
@@ -15,32 +14,25 @@ object VmSandbox {
                            pExecutor: Address)
       extends Environment {
 
-    val balances = mutable.Map(initBalances: _*)
-    val programs = mutable.Map(initPrograms: _*)
-    val sealedPrograms = mutable.Map[Address, Boolean]()
+    private val balances = mutable.Map(initBalances: _*)
```

Comment:
why?

Reply:
Why not?

Reply:
It's possible that in future we will need to access these fields.

Reply:
It's a bad practice to expose mutable state to public.

Reply:
If we will need to access this data we should add a method.